EIPS/eip-1815.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 1815
+title: Interface for blind auctions
+author: Albert Acebrón (@corollari)
+discussions-to: https://github.com/ethereum/EIPs/issues/1745
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-03-04
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Standard interface for contracts implementing blind auctions.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+Currently there doesn't exist any standard interface/contract for blind auctions, creating a void that has been filled with many de-facto standards such as the one used by ENS or the one used in the blind auction contract present in Solidity's documentation. This ERC aims to remove this fragmentation by proposing a standard interface for blind auction contracts.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+Many dApps that use blind auctions have appeared in the last years, each one reinventing the wheel and implementing blind auctions in ways that are conceptually similar but with special quirks and different interfaces, thus making it impossible for a system to interact with all of these without accommodating all these differences. This ERC aims to solve that by standarizing an interface for blind auctions contracts, which will enable the creation of systems that can interact with any blind auctions without requiring specialized code.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+```
+interface BlindAuction {
+    enum Mode { Open, Auction, Owned, Forbidden, Reveal, NotYetAvailable }
+
+    event AuctionStarted(bytes32 indexed hash, uint registrationDate);
```

Comment:
I think it would be a good idea to comment (perhaps using NatSpec) each of these interfaces as done in other ERCs. See http://eips.ethereum.org/EIPS/eip-721 as an example.

Reply:
Will start working on it